README.md

```diff
@@ -51,6 +51,10 @@ one of:
 - success (200, JSON "status": "OK")
 - failure (500, JSON "status": "error").
 
+### Kafka scripts
+
+- https://github.com/ONSdigital/dp-data-tools
```

Comment:
It would be good to use the markdown link syntax rather than just a bare link. Also it might be useful to have some description here to clarify that the scripts might help with issues rather than them being required for any kind of setup.